### PR TITLE
[Translation zh-tw] Fundamentals > Payments

### PR DESCRIPTION
**Pull request by kennith**
What's changed, or what was fixed?

Remove a broken link.

**CC:** @petele

### CODE REVIEW

**Comment by lukechi1219** [comment]
@sotayamashita Yes, I've signed in the CLA.

image

**Comment by limhenry** [comment]
@lukechi1219 Quick update: can you change *Pay with Google* to *Google Pay*?

File: src/content/zh-tw/fundamentals/payments/index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+![](images/4_the_payment_transaction_process.png)
+
+*付款交易流程*
+
+流程開始於，當商家網站創建一個新的 `PaymentRequest` 並傳遞給瀏覽器所需的所有購買資訊：付款金額，期望付款的貨幣，以及網站接受的付款方式。瀏覽器會判斷網站接受的付款方式與使用者在目標設備上安裝的付款方式是否彼此相容。
+
+然後瀏覽器會將付款 UI 顯示給使用者看，讓使用者選擇付款方式並授權交易。付款方式可以非常簡單例如使用瀏覽器已儲存的信用卡資料，也可以深奧複雜例如另外呼叫專門為處理網站付款而編寫的第三方應用程式。
+
+註：您可以用 "Pay with Google" 此付款方式從使用者的 Google 帳戶取得卡片資訊和取得設備上的付款 tokens 。要了解更多資訊，請閱讀 [Google Pay API 文件](/pay/api/web/paymentrequest/tutorial)。
```
Comment (kennith) [inline src/content/zh-tw/fundamentals/payments/index.md]: Change to Google Pay

**Comment by kennith** [comment]
@limhenry the cla are not accepted.

File: src/content/zh-tw/fundamentals/payments/index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+project_path: /web/_project.yaml
+
+book_path: /web/fundamentals/_book.yaml
+
+description：Payment Request API 是用於在網路上進行快速，簡單的付款。
```
Comment (petele) [inline src/content/zh-tw/fundamentals/payments/index.md]: Uses a unicode fullwidth colon, please change to normal colon.

File: src/content/zh-tw/fundamentals/payments/index.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+# Payment Request (付款要求) API 介紹 {: .page-title }
+
+{% include "web/_shared/contributors/agektmr.html" %}
+
+{% include "web/_shared/contributors/dgash.html" %}
+
+{% include "web/_shared/contributors/zkoch.html" %}
+
+在網路上購買商品是一種方便但經常令人沮喪的體驗，特別是在行動裝置上。雖然行動裝置流量繼續增加，但行動裝置轉化僅佔所有已完成購買的三分之一。換句話說，使用者在行動裝置上放棄購買的數量，是在桌機 (非行動裝置) 上放棄購買的兩倍 (前提：行動裝置跟桌機的使用者流量大約相同)。為什麼？
```
Comment (petele) [inline src/content/zh-tw/fundamentals/payments/index.md]: Uses a unicode fullwidth colon, please change to normal colon.

File: src/content/zh-tw/fundamentals/payments/index.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+導致放棄購買的大多數問題可以直接追溯到購買表單。每個應用程式或網站都有自己的數據輸入和驗證流程，使用者通常會發現他們必須在每個應用程式的購買畫面輸入相同的資訊。此外，應用程式開發人員必須費盡心力 (struggle 掙扎) 建立購買流程，才能讓程式或網站支援多種不同的付款方式；即使付款方式要求的細微差異也會使填寫購買表單和送出表單的流程複雜化。
+
+如果有任何系統能改進或解決這些問題中的一個或多個部分，所有人都會樂於接受改變。我們已經開始使用 [Autofill](/web/updates/2015/06/checkout-faster-with-autofill) 解決問題，但現在我們想談一個更全面的解決方案。
+
+## Payment Request API 介紹 {: #introducing}
+
+Payment Request API 是 [W3C標準](https://www.w3.org/TR/payment-request/) 候選人，旨在*消除結帳表單* 。它極大地改善了購買過程中使用者的操作流程，提供了更加一致的使用者體驗，並使網路商家能夠輕鬆地提供不同的付款方式。
+
+Payment Request API 被設計成與供應商無關 (vendor-agnostic)，意味著它不需要使用特定的付款系統。這不是一種新的付款方式，也不是直接與付款處理系統串接整合 (integrate)；相反地，它是用來將使用者的付款和送貨資訊傳送到網路商家，具有以下目標：
```
Comment (petele) [inline src/content/zh-tw/fundamentals/payments/index.md]: Uses a unicode fullwidth colon, please change to normal colon.

File: src/content/zh-tw/fundamentals/payments/index.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+### 付款交易流程{: ＃transaction-process}
+
+對使用者和網路商家來說，使用 Payment Request API 可以讓交易流程盡可能地無縫順暢 (seamless)。
+
+![](images/4_the_payment_transaction_process.png)
+
+*付款交易流程*
+
+流程開始於，當商家網站創建一個新的 `PaymentRequest` 並傳遞給瀏覽器所需的所有購買資訊：付款金額，期望付款的貨幣，以及網站接受的付款方式。瀏覽器會判斷網站接受的付款方式與使用者在目標設備上安裝的付款方式是否彼此相容。
```
Comment (petele) [inline src/content/zh-tw/fundamentals/payments/index.md]: Uses a unicode fullwidth colon, please change to normal colon.

File: src/content/zh-tw/fundamentals/payments/index.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+然後瀏覽器會將付款 UI 顯示給使用者看，讓使用者選擇付款方式並授權交易。付款方式可以非常簡單例如使用瀏覽器已儲存的信用卡資料，也可以深奧複雜例如另外呼叫專門為處理網站付款而編寫的第三方應用程式。
+
+註：您可以用 "Google Pay" 此付款方式從使用者的 Google 帳戶取得卡片資訊和取得設備上的付款 tokens 。要了解更多資訊，請閱讀 [Google Pay API 文件](/pay/api/web/paymentrequest/tutorial)。
```
Comment (petele) [inline src/content/zh-tw/fundamentals/payments/index.md]: Uses a unicode fullwidth colon, please change to normal colon.

File: src/content/zh-tw/fundamentals/payments/index.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+    <img src="images/5_9_payment_request_ui.png">
+    <figcaption>Payment Request 介面</figcaption>
+  </figure>
+</div>
+
+在使用者授權交易後，所有必要的付款細節將直接發送回該網站。例如，對於信用卡付款，網站將取回卡號，持卡人姓名，到期日期和 CVC。
+
+Payment Request 還可以擴展用來返回其他資訊，例如送貨地址和選項、付款人電子郵件、和付款人電話。這樣您就可以獲得完成付款所需的所有資訊，從而無需向使用者顯示結帳表單。
+
+新流程的美麗之處有三個層面：從使用者的角度來看，以前所有乏味的互動流程：付款要求、授權、進行付款和結果，現在都在一個步驟中進行；從網站的角度來看，它只需要呼叫一次 JavaScript API；從付款方式的角度來看，沒有任何流程改變。
```
Comment (petele) [inline src/content/zh-tw/fundamentals/payments/index.md]: Uses a unicode fullwidth colon, please change to normal colon.

File: src/content/zh-tw/fundamentals/payments/index.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
+project_path: /web/_project.yaml
+
```
Comment (petele) [inline src/content/zh-tw/fundamentals/payments/index.md]: Don't leave blank lines between the yaml front matter

**Pull request merged**